src/app/child-dev-project/children/child-details/child-details.component.html

```diff
@@ -76,7 +76,7 @@ <h1 class="page-header section-child">
 
         <div fxFlex>
           <mat-form-field style='width: 300px;'>
-            <input matInput formControlName="name" placeholder="Name" title="name" type="text">
+            <input matInput formControlName="name" placeholder="Name" required="true" title="name" type="text">
```

Comment:
I'm reverting this as it doesn't seem necessary for the bugfix and to keep definition of required fields in one place only (in the initForm() of child-details.component.ts)